environment/src/main/java/com/sequenceiq/environment/resourcepersister/Resource.java

```diff
@@ -1,30 +1,33 @@
 package com.sequenceiq.environment.resourcepersister;
 
-import java.io.Serializable;
-
 import javax.persistence.Column;
 import javax.persistence.Convert;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 import javax.persistence.SequenceGenerator;
 
-import com.sequenceiq.cloudbreak.common.json.Json;
-import com.sequenceiq.cloudbreak.common.json.JsonToString;
 import com.sequenceiq.cloudbreak.converter.CommonStatusConverter;
 import com.sequenceiq.common.api.type.CommonStatus;
+import com.sequenceiq.common.api.type.ResourceType;
+import com.sequenceiq.environment.environment.domain.Environment;
 import com.sequenceiq.environment.parameters.dao.converter.ResourceTypeConverter;
 
 @Entity
-public class Resource implements Serializable {
+public class Resource {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO, generator = "resource_generator")
     @SequenceGenerator(name = "resource_generator", sequenceName = "resource_id_seq", allocationSize = 1)
     private Long id;
 
-    private String instanceGroup;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "resource_environment")
```

Comment:
if you would name the column in your sql `environment_id` I think you could drop this line